scripts/test-dynamic-model.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  completion,
+  enqueueAddModel,
+  pollForJobCompletion,
+  runIfCalledAsScript
+} from 'scripts/shared';
+
+async function main() {
+  const modelName = 'microsoft/Orca-2-13b';
```

Comment:
can we do phi-2? (if this test is still helpful for small models)

it has a better license

Reply:
Yeah that'd be great IMO. We should prob read the model name from env as well, so that we can quickly test for new download :-?

Would be great to also use it as the default model in general.